docs/cloud-tutorials/application-management.md

```diff
@@ -57,6 +57,13 @@ If you edit your application or schema, run `npx dbos-cloud app deploy` again to
 You don't have to worry about changing database server connection parameters like `hostname` or `password` in [`dbos-config.yaml`](../api-reference/configuration.md) to deploy an application to the cloud&#8212;DBOS automatically applies the connection information of your cloud database instance.
 :::
 
+When you deploy a new version of your application to DBOS cloud, currently running workflows will still run to completion. This means database schema migrations must be backward compatible with these running workflows.
```

Comment:
Same comment as below, set this off into an infobox with more explanation.

Reply:
Should be addressed in both cases